src/Authentication/PrivateRoute.js

```diff
@@ -0,0 +1,35 @@
+import React, { useContext } from "react";
+import { Route, Redirect } from "react-router-dom";
+import { AuthContext } from "./Auth";
+import Navigation from "../Components/Navigation/Navigation";
+import Header2 from "../Components/Header/Header2";
+
+const PrivateRoute = ({ component: RouteComponent, ...rest }) => {
+  const {currentUser} = useContext(AuthContext);
+  return (
+
+    !!currentUser ? (

```

Comment:
what's the point of the double bang (!!) here? We're not assigning a bool to a value so I think you can get away with just `currentUser ?...`

Reply:
!! is like the double vs triple equal 

It converts an object into a boolean